docs/painless/painless-contexts/painless-context-examples.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[painless-context-examples]]
+=== Context examples
+
+Each example must be run in the order of the table in the
+<<painless-contexts, previous section>> to work correctly.  To run each example,
+first index the sample data into Elasticsearch:
+
```

Comment:
How about, "To run the examples, you need to index the sample seat data into Elasticsearch. The examples must be run sequentially to work correctly."

Reply:
Done.